src/index.ts

```diff
@@ -394,5 +394,5 @@ export const parsePermissionData = (data: Buffer): PermissionData => {
 
 export { PythConnection } from './PythConnection'
```

Comment:
Can you also update `example_ws_usage.ts` to import these items from `"." `

Reply:
Good call! Also updated https example